app/views/posts/index.html.erb

```diff
@@ -0,0 +1,5 @@
+<div>
+  <% @posts.each do |post| %>
+    <%= post.title %><br>
+  <% end %>
+</div>
```

Comment:
- [ ] - [ ] Same here, Please add a placeholder written in plain HTML explaining what this view suppose to show.